modin/core/io/pickle/pickle_dispatcher.py

```diff
@@ -89,3 +92,50 @@ def _read(cls, filepath_or_buffer, **kwargs):
         return cls.query_compiler_cls(
             cls.frame_cls(partition_ids, new_index, new_columns)
         )
+
+    @classmethod
+    def _to_method(cls, qc, **kwargs):
```

Comment:
why does the name of the method so strange?
```suggestion
    def _to_pickle(cls, qc, **kwargs):
```

Reply:
My guess that it should be a generic name like "_read" for any read function so I definitely would not want to add the format name. Maybe just `_to`?

Reply:
maybe something like `export()`?

by the way, why does the method's name starts with `_`, the `.read` name doesn't do so

Reply:
`write` looks good.

`read` is a base class method (`FileDispatcher`) that calls the `_read` method of the child class. As long as there is no such structure for write functions, methods can be called without `_`, ok.